partcad/src/partcad/actions/part_actions.py

```
@@ -0,0 +1,77 @@
+    target_path = Path(project.path) / f"{name}.{kind}"  # Define the target file path
+
+    logging.info(f"Importing the part {name} of type {kind} from {source_path} to {target_path}")
+
+    # Check if the source file exists
+    if not source_path.exists():
+        raise ValueError(f"Source file '{source_path}' does not exist.")
+
+    # Copy the file to the project directory
+    try:
+        shutil.copy(source_path, target_path)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)